tests/auth_tests/test_auth_backends.py

```diff
@@ -213,6 +216,31 @@ def create_users(self):
             password='test',
         )
 
+    def test_authenticate_inactive(self):
+        """
+        To verify that inactive user being rejected.
+        """
+        self.user.is_active = False
+        self.user.save()
+        self.assertIsNone(
+            authenticate(
+                username='test',
```

Comment:
to ensure the a typo in the credentials isn't the reason for the failure, you could create a dict with them and verify authentication is successful before you set `self.user.is_active = False`. e.g. `self.assertEqual(authenticate(**credentials), self.user)`